docs/gameplay-features/titan-tools/titan-axe.md

```diff
@@ -1,3 +1,30 @@
 # Titan Axe
 
-![](<../.gitbook/assets/Titan Axe.PNG>)
+![](<../../.gitbook/assets/The Titan Axe.png>)
+
+  ### There are 3 types of Titan Axe here are the prices
+  
+  - The Red Titan Axe 8 Jigsaws
```

Comment:
Thoughts on whether we should include the prices here or let people know to check the prices at /warp titan?

I worry if there's ever a price update that we might forget to update this part of the docs.